scripts/setup/rewrite_yaml_files.sh

```diff
@@ -74,6 +62,18 @@ cat serving-core.yaml |
         )
     )' |
     yq '
+    (
```

Comment:
Previous section deletes the affinity. Can you make it into a single assignment instead of cleaning and appending the affinity field?